### ISSUE TITLE
Checkout failing to load after theme upgrade

### ISSUE THREAD

**Issue by king-puddles** [bigcommerce/checkout-js#1287]
**Describe the bug**
TLDR; Credit Card fields not loading after changing to an updated theme - Braintree gateway

Details
We (in house) have recently rebuilt our website using the latest (2.8.0) "Peak" theme code base. We have noticed however that users are unable to enter credit card details in the credit card payment option. We are using the Optimized One-Page Checkout and have made no customisations to checkout functionality (as far as we're aware).
It seems the page is failing to load the required scripts for one reason or another, disabling the fields. Depending on the browser depends on the errors thrown. See below for Firefox errors.

We also have similar behaviour in testing, however we simply receive 403 errors for hostedFields.js and client.js from Braintree. This may simply be a HTTP issue.

Reverting to the old version of the theme (based on Peak 1.10) seems to resolve the issue. What is also interesting is that even applying the stock Cornerstone theme still has the same issue.

Are we missing something with newer themes that would be causing the checkout to break? 
We have contacted BC support directly who closed our case almost immediately. We have also been in discussion with Braintree, however they did not develop the checkout page - fair enough.

Note that this issue seems to only apply to the credit card payment option - it seems all other payment options work as expected.

**To Reproduce**
1. Apply newer updated them (Peak or Cornerstone)
2. Go to checkout page
3. Get to payment option step
4. Notice you are unable to enter credit card details
5. Apply older theme used and repeat steps above to work as expected

**Expected behavior**
Checkout page to correctly load all scripts and allow credit card entry.

**Screenshots**
<img width="1435" alt="Live" src="https://github.com/bigcommerce/checkout-sdk-js/assets/44661030/b7c3b5e3-496b-47dc-b577-788dc7af4f2e">

**Desktop (please complete the following information):**
 - OS: Various
 - Browser: Various

**Comment by animesh1987** [comment]
Hey @king-puddles do you know what version of checkout the store above is using?

**Comment by king-puddles** [comment]
@animesh1987 I have tested the store using the latest Cornerstone theme - how can I confirm what version of the checkout is being used?

**Comment by animesh1987** [comment]
@king-puddles  Just to check, the issue is reproducible on a store using the following setup
- native checkout (no customisations)
- base cornerstone theme 

Also will be good if you can share the store url. Thanks

**Comment by king-puddles** [comment]
@animesh1987 Yes, from what I have seen so far this is correct.
Our url is https://www.minizoo.com.au

Please note we are currently using our old theme as this seems to work for some reason and we are running a promotion so payments going through is a must.

**Comment by animesh1987** [comment]
Thanks @king-puddles that does point to some integration issues between the theme and checkout/braintree form setup.

I would suggest to contact support in this case, as they can try and replicate the issue on a sandbox with the theme which will help engineers at our end to debug the issue. 

Also if you do have a sandbox with the theme with issue applied that might help as well.

**Comment by king-puddles** [comment]
Thanks @animesh1987 for the help. Can I just confirm you were able to replicate the issue or at least maybe see something wrong?

I have an open ticket with support and have been keeping them updated.

I'm not sure I have access to the sandbox environment (I assume you are talking about the developer program) as I am simply the "developer" for our BC store and not a wider client base - this might be worth applying for anyway and see if I can get something set up.

**Comment by king-puddles** [comment]
Updating thread for closure.

Looks like the newer themes do in fact have a small template file for the checkout where the older theme doesn't. 
This template includes a reference to load in footer scripts which was pulling in some erroneous code.

We've updated and moved the scripts to the new script manager and all is well in our world.
